go/kernel/linux/eventfd.go

```diff
@@ -0,0 +1,27 @@
+package linux
+
+import (
+	"syscall"
+)
+
+func (k *LinuxKernel) Eventfd2(initval, flags uint) uint64 {
+	var a3 uintptr
+	r1, _, err := syscall.Syscall(syscall.SYS_EVENTFD2, uintptr(initval), 
+uintptr(flags), a3)
+	if err != 0 {
+		panic(err)
```

Comment:
Errors are okay to return to the program. Some programs need an error to tell them what's up. Please don't panic in syscalls, and remember you can use `usercorn -strace` to diagnose them.

You should be able to return `posix.Errno(err)` in most Linux error cases (import `github.com/lunixbochs/usercorn/go/kernel/posix`)